api/v1alpha1/addonimageset_types.go

```diff
@@ -38,16 +38,16 @@ type AddonImageSetSpec struct {
 
 	// +optional
 	// OCM representation of an add-on parameter
-	AddOnParameters *ocmv1.AddOnParameterList `json:"addOnParameters"`
+	AddOnParameters *[]ocmv1.AddOnParameter `json:"addOnParameters"`
```

Comment:
I like this simplification. I thought we needed to have an `AddOnParameterList` with a `DeepCopy()` method, but saw your `copy(...)` code. Nice job ! You've officially become my go-to golang specialist. :heart: